examples/pytorch/summarization/run_summarization_no_trainer.py

```diff
@@ -164,11 +164,8 @@ def parse_args():
     parser.add_argument(
         "--max_length",
         type=int,
-        default=128,
-        help=(
-            "The maximum total input sequence length after tokenization. Sequences longer than this will be truncated,"
-            " sequences shorter will be padded if `--pad_to_max_lengh` is passed."
-        ),
+        default=None,
+        help=("The maximum target length to use when predicting with the generate method."),
```

Comment:
This is the maximum length for the inputs, not the targets. I don't think this should be touched.

Reply:
There is another argument for the inputs `max_source_length`. So I suppose the `max_length` is used to constrain the generation length

Reply:
It's actually never used if I look more closely, so let's remove it entirely.